### PR TITLE
ChatGPT Case Study

### PR DESCRIPTION
**Pull request by jonaslod**
Development Platforms CA

### CODE REVIEW

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+- November 2019: OpenAI launches GPT-2 as a "direct scale-up" of GPT.
+
+- July 2020: GPT-3 is introduced in beta testing mode.
+
+- 29th June 2021: GitHub releases GitHub Copilot, developed in collaboration with OpenAI and based on GPT-3.
+
+- 5th August 2022: Meta launches Blenderbot 3, the company's most advanced AI chatbot to date.
+
+- 30th November 2022: OpenAI launches ChatGPT based on the GPT-3.5 model. It gained 1 million users within 5 days of its release, whereas other successful applications needed months to achieve the first million users. For instance, Netflix achieved its first million in 40 months, Facebook in 10 months, and Instagram in 2.5 months.
+
```
[the remark this answers is not in the document]
Comment (Fermain) [reply case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:49]: There have been a rush of releases in this sector recently. AutoGPT, BabyAGI, JARVIS/HuggingGPT

There have also been releases from Facebook (LLaMa), Google (Bard/LAMDA) and even open source models.

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+ChatGPT is currently free to use. OpenAI has announced on [Twitter](https://twitter.com) that it's piloting a subscription plan that offers faster response times and reliability during peak hours. At the time of writing this report, the plan costs $20/month. The free tier will still be available to the public.
```
Comment (Fermain) [inline case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:73]: How about API access to GPT?

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+### How it works
+ChatGPT uses a deep neural network architecture called a Transformer, which was introduced in 2017. Transformers are designed to process sequential data, such as text, and are particularly well-suited to generating text that is coherent and grammatically correct. Since it is trained using a large amount of text data, it learns the statistical patterns and relationships that exist within human language. When a user inputs a prompt, ChatGPT uses this knowledge to generate a response that is relevant.
```
Comment (Fermain) [inline case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:76]: It is worth mentioning that due to the sheer size and complexity of the algorithm, exactly how it works cannot be known

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+ChatGPT can be fine-tuned on specific tasks and datasets to improve its performance for particular use cases. This means that organisations can train ChatGPT to specialise in certain types of writing, such as legal or medical writing, or to incorporate domain-specific language and terminology.
+
+
+## Weaknesses
+
+Despite its many possible uses and strengths, this tool comes with a series of limitations that are important to take into account. One critical issue is its lack of factual correctness at times. The bot may provide answers that are factually incorrect, but in such an assertive manner it could be difficult for an untrained person to realise. Moreover, it gives no references on where it took the data from. In order to avoid the spread of misinformation, users should double check the answers by consulting reliable sources of information in the subject matter. Additionally, it tends to be unpredictable and unstable. This means that users may avoid content moderation by repeatedly asking the same question in different ways.
```
Comment (Fermain) [inline case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:96]: This is known as `hallucination`

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+ChatGPT is one of several NLP models that are capable of generating human-like text. Some of the other models that are commonly used for this purpose include Google's Meena, Microsoft's XiaoIce, and Facebook's Blender. Compared to these models, ChatGPT is generally considered to be more versatile and capable of generating text in a wider range of styles and genres. However, it also tends to generate more irrelevant or nonsensical responses than some of the other models.
```
Comment (Fermain) [inline case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:115]: This could be updated for 2023 with Lamda, Llama, Alpaca, etc.

**Review by Fermain — CHANGES_REQUESTED** [review]
No major issues but needs some refreshing in some places.

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonaslod) [reply case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:49]: Done

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md (hunk 2 above)
Comment (jonaslod) [reply case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:83]: Done

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md (hunk 3 above)
Comment (jonaslod) [reply case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md:91]: Done

File: case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md (hunk 4 above)
Comment (jonaslod) [reply case-studies/openai/chatgpt/chatGPT-case-study-jonaslod.md]: Done

**Review by Fermain — APPROVED** [review]

**Pull request merged**